app/models/foreman_google/google_compute.rb

```diff
@@ -1,15 +1,24 @@
 module ForemanGoogle
   class GoogleCompute
+    attr_reader :identity, :status
+
+    def initialize(service:, zone:, identity: nil)
+      @service = service
```

Comment:
If we pass `client` variable in `find_vm_by_uuid` method to the `service` arg, can we rename the arg to the `client`?
I'm kinda confused from the `service: client`, `service: service` or `client: client` looks better to me